specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2023-02-01-preview/containerInstance.json

```diff
@@ -0,0 +1,2471 @@
+          "type": "integer",
+          "format": "int32"
+        },
+        "successThreshold": {
+          "description": "The success threshold.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "timeoutSeconds": {
+          "description": "The timeout seconds.",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "ContainerGroup": {
+      "description": "A container group.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "$ref": "#/definitions/ContainerGroupProperties"
+        }
+      ]
+    },
+    "ContainerGroupProperties": {
+      "description": "The container group properties",
+      "type": "object",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/ContainerGroupIdentity",
+          "description": "The identity of the container group, if configured."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "type": "object",
+          "description": "The container group properties",
+          "properties": {
+            "provisioningState": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The provisioning state of the container group. This only appears in the response."
+            },
+            "containers": {
+              "type": "array",
+              "description": "The containers within the container group.",
+              "items": {
+                "$ref": "#/definitions/Container"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            },
+            "imageRegistryCredentials": {
+              "type": "array",
+              "description": "The image registry credentials by which the container group is created from.",
+              "items": {
+                "$ref": "#/definitions/ImageRegistryCredential"
+              },
+              "x-ms-identifiers": [
+                "username"
+              ]
+            },
+            "provisioningTimeoutInSeconds": {
+              "type": "integer",
+              "description": "Time in seconds in which a container group deployment would timeout and fail. The allowed maximum value is 1800 seconds. If value is not provided, property is given maximum value by default.",
+              "format": "int32"
+            },
+            "isCustomProvisioningTimeout": {
+              "type": "string",
+              "readOnly": true,
+              "description": "Flag indicating whether a custom value was provided for the provisioningTimeoutInSeconds property",
+              "enum": [
+                "True",
+                "False"
+              ],
+              "x-ms-enum": {
+                "name": "IsCustomProvisioningTimeout",
+                "modelAsString": true
```

Comment:
I have already set modelAsString to true in the previous commit. Also the documentation indicates that it's set to true as default.